api/net/Areas/Admin/Controllers/ReportController.cs

```diff
@@ -217,7 +217,7 @@ public async Task<IActionResult> SendToAsync(int id, string to)
         var username = User.GetUsername() ?? throw new NotAuthorizedException("Username is missing");
         var user = _userService.FindByUsername(username) ?? throw new NotAuthorizedException($"User [{username}] does not exist");
 
-        var request = new ReportRequestModel(ReportDestination.ReportingService, Entities.ReportType.Content, report.Id, new { })
+        var request = new ReportRequestModel(ReportDestination.ReportingService, Entities.ReportType.Content, report.Id, JsonDocument.Parse("{}"))
```

Comment:
Needed to change to JsonDocument because it was being used inconsistently with a dynamic object.